src/components/BannerFallback/HalvingBanner.tsx

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react'
+import classnames from 'classnames'
+import styles from './index.module.scss'
+import SimpleButton from '../SimpleButton'
+import { useAppState } from '../../contexts/providers'
+import { useCountdown } from '../../utils/hook'
+
+export const HalvingBanner = () => {
+  const { statistics } = useAppState()
+
+  const HALVING_EPOCH = 8760
+  const currentEpoch = Number(statistics.epochInfo.epochNumber)
+  const NextHalvingCount = Math.ceil(currentEpoch / HALVING_EPOCH) + 1
+  const targetEpoch = HALVING_EPOCH * NextHalvingCount
+  const singleEpochAverageTime = Number(statistics.estimatedEpochTime)
+  const currentEpochUsedTime =
+    (Number(statistics.epochInfo.index) / Number(statistics.epochInfo.epochLength)) * singleEpochAverageTime
+
+  const estimatedTime = (targetEpoch - currentEpoch) * singleEpochAverageTime - currentEpochUsedTime
+
+  const estimatedDate = useMemo(() => new Date(new Date().getTime() + estimatedTime), [])
+  const [days, hours, minutes, seconds] = useCountdown(estimatedDate)
+
+  const count = () => {
+    if (days > 0) {
+      return <div className={styles.HalvingBannerCount}>{days} Days</div>
+    }
+    if (hours > 0) {
+      return <div className={styles.HalvingBannerCount}>{hours} Hours</div>
+    }
+    if (minutes > 0) {
+      return <div className={styles.HalvingBannerCount}>{minutes} Minutes</div>
+    }
+    if (seconds > 0) {
+      return <div className={styles.HalvingBannerCount}>{seconds} Seconds</div>
+    }
+
+    return <div className={styles.HalvingBannerCount}> Halving! </div>
```

Comment:
we could add a `timeIsDue` field in the return object of `useCountdown` hook, if its true, render `Halving!`